core/domain/skill_services.py

```diff
@@ -1065,6 +1065,34 @@ def skill_has_associated_questions(skill_id):
     return len(question_ids) > 0
 
 
+def filtered_skills_with_none_type_keys(degrees_of_mastery):
+    """Sort the dict based on the mastery value.
+
+    Args:
+        degrees_of_mastery: dict(str, float|None). Dict mapping
+            skill ids to mastery level.
```

Comment:
Maybe just add a sentence that the mastery level can be `None`, even though it is mentioned in the type it would be worth mentioning.

Reply:
Added